common/changes/@itwin/geonames-extension/node-types-18_2022-09-14-15-18.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
merge with master, then you should be able to nuke this file

Reply:
It's already merged and up-to-date with master, which is why I am confused by it. This change file doesn't exist in master. Perhaps the changes were reverted but the change file still stayed in my branch?

Reply:
try deleting the file, the pkg doesnt exist anymore, so you should be able to pass `rush change -v` w/o this

Reply:
Everything I've tried with `rush change` still doesn't pass. The error is that it can't find the change file that I am trying to delete..

Reply:
I deleted it and pushed.